packages/auth/demo/src/index.js

```diff
@@ -277,7 +277,7 @@ function onAuthError(error) {
     // Handle second factor sign-in.
     handleMultiFactorSignIn(getMultiFactorResolver(auth, error));
   } else {
-    alertError('Error: ' + error.code);
+    alertError('Error: ' + error.code + 'user email : ' + error.customData.email);
```

Comment:
Not every error will have a customData.email field - maybe we just leave it as the error code for now?

Reply:
I added this, so we can identify other error paths where the email is missing but should be propagated, during testing with the demo app.

Reply:
Right, what I mean is not every user that is signed up with Firebase Auth will have an email attached to them. For finding error paths where the email is missing, maybe we can leave that as a local change while we're developing

Reply:
Sounds good, done.